pkg/otlp/attributes/attributes_translator.go

```diff
@@ -0,0 +1,61 @@
+// Copyright  OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package attributes
+
+import (
+	"context"
+	"fmt"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+
+	"github.com/DataDog/opentelemetry-mapping-go/pkg/otlp/attributes/source"
+)
+
+const missingSourceMetricName string = "datadog.otlp_translator.attributes.missing_source"
```

Comment:
```suggestion
const missingSourceMetricName string = "datadog.otlp_translator.resources.missing_source"
```

Technically we are counting the number of resources without host identifying attributes, rather than attributes ?

Reply:
(Assuming the logs code gets refactored)